styles.css

```diff
@@ -199,44 +199,53 @@ a {
   gap: 2rem;
   font-size: 15px;
   font-weight: 500;
-  width: 0px;
+  width: 0;
 }
-  
-.menu-mobil{
+
+.menu-mobil {
```

Comment:
- [ ] Good job so far 👍 your menu works as expected but I have a small issue with the design 😢  Please make sure that your mobile menu design is similar to the figma template. Your **Xmark** and menu option spacing are a bit different, please check the screenshots below ⬇️  to see the differences and fix them 👍 

#### You mobile menu design
![menuAUTHOR](https://user-images.githubusercontent.com/6615394/223527056-774628db-c7ce-40ac-87c1-31896b56e61f.jpg)


#### figma template
![menuuuuuu](https://user-images.githubusercontent.com/6615394/223527064-27a269b8-12d5-4070-b86f-f56dc771e0e1.PNG)